routes/chorals.js

```diff
@@ -141,8 +141,6 @@ router.get('/:choralId', function(req, res, next) {
     Choral.findOne({ choralId: choralId }, (err, choral) => {
       if (err || !choral) {
         logHelper.createLog("error", 'Choral does not exist: ' + err, ["chorals", "get"]);
```

Comment:
I think it's reasonable to just show a 404 if someone types .../chorals/1234 but I guess we could just show an error page thing is if we don't even have it in mongo, i have 0 information to populate the page with